### PR TITLE
built-mobile-version

### PR DESCRIPTION
**Pull request by ONJoseph**
built the mobile version

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
> built the mobile version

- [ ] Kindly make your pull request message more descriptive explaining more of what you did in the pull request

**Comment by ONJoseph** [comment]
made pull request message more descriptive explaining more of what I did in the pull request as suggested by code reviewer.

**Comment by ONJoseph** [comment]
> > built the mobile version
> 
>     * [x]  Kindly make your pull request message more descriptive explaining more of what you did in the pull request

Suggestion implemented. Thanks.

**Review by prolajumokeoni — COMMENTED** [review]
Hi @ONJoseph , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the previously approved milestone, kindly finish the first milestone before starting the second
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by prolajumokeoni** [comment]
> > > built the mobile version
> > 
> > 
> > ```
> > * [x]  Kindly make your pull request message more descriptive explaining more of what you did in the pull request
> > ```
> 
> Suggestion implemented. Thanks.

I can not review your project because the previous milestone is yet to be approved,  you need to finish a milestone before moving to the next,  your code reviews are still intact so do well to submit for the second milestone when the first is approved, keep rocking 💪🏿

**Comment by prolajumokeoni** [comment]
> made pull request message more descriptive explaining more of what I did in the pull request as suggested by code reviewer.

Hi @ONJoseph your pull request title is great, it's your pull request message you need to make more descriptive what you wrote currently is "built the mobile version"

**Comment by ONJoseph** [comment]
> > > > built the mobile version
> > > 
> > > 
> > > ```
> > > * [x]  Kindly make your pull request message more descriptive explaining more of what you did in the pull request
> > > ```
> > 
> > 
> > Suggestion implemented. Thanks.
> 
> I can not review your project because the previous milestone is yet to be approved, you need to finish a milestone before moving to the next, your code reviews are still intact so do well to submit for the second milestone when the first is approved, keep rocking 💪🏿

Ok. Thanks a lot @prolajumokeoni

File: index.html [hunk 1]
```diff
@@ -20,10 +20,11 @@
...
+            <img src="./src/Icons/Union.png" alt="">
             <ul class="menu-item">
               <li id="line" class="line-1"><a href="#"></a></li>
-              <li id="line" class="line-2"><a href="#"></a></li>
-              <li id="line" class="line-3"><a href="#"></a></li>
+              <li id="line-2" class="line-2"><a href="#"></a></li>
+              <li id="line-3" class="line-3"><a href="#"></a></li>
               <li class="menu-links">Portfolio</li>
               <li class="menu-links">About</li>
               <li class="menu-links">Contact</li>
```
Comment (ShoiraTa) [inline index.html:30]: - [ ] Please make sure that in mobile view, the header has a humburger button on the right side, currently I can see a list of 3 menu items: 

Your design  | Figma design
------------- | -------------
Screenshot 2022-02-19 at 21 03 11  | Screenshot 2022-02-19 at 21 03 51

File: index.html [hunk 2]
```diff
@@ -57,8 +58,313 @@ <h6 class="link-item">
...
+            </h6>
+          </div>
+          <div class="network-container">
+            <ul class="network-items">
+              <li class="icon"><a class="picture twitter" href="#"></a></li>
+              <li class="icon"><a class="picture linkedlin" href="#"></a></li>
+              <li class="icon"><a class="picture medium" href="#"></a></li>
+              <li class="icon"><a class="picture github" href="#"></a></li>
+              <li class="icon"><a class="picture angellist" href="#"></a></li>
+            </ul>
```
Comment (ShoiraTa) [inline index.html:262]: - [ ] Please make sure that all social icons are visible and the` let's connect` font style is the same:

Your design  | Figma design
------------- | ------------- 
Screenshot 2022-02-19 at 21 07 09 | Screenshot 2022-02-19 at 21 06 52

File: index.html [hunk 3]
```diff
@@ -57,8 +58,313 @@ <h6 class="link-item">
               <li class="icon"><a class="picture linkedlin" href="#"></a></li>
               <li class="icon"><a class="picture medium" href="#"></a></li>
               <li class="icon"><a class="picture github" href="#"></a></li>
-              <li class="icon"><a class="picture peace" href="#"></a></li>
+              <li class="icon"><a class="picture angellist" href="#"></a></li>
```
Comment (ShoiraTa) [inline index.html:61]: - [ ] Please make sure that all social icons are visible and the` let's connect` font style is the same:

Your design  | Figma design
------------- | ------------- 
Screenshot 2022-02-19 at 21 07 09 | Screenshot 2022-02-19 at 21 06 52

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ONJoseph,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## OPTIONAL 

- [ ] I recommend you making the pull request more descriptive 😉 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my name if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ONJoseph) [reply index.html:30]: change implemented. But can't access my student dashboard, as I just lost all my tabs. Will submit it for review as soon as I gain access to my dashboard. Thanks

File: index.html (hunk 2 above)
Comment (ONJoseph) [reply index.html:262]: Change implemented

File: index.html (hunk 1 above)
Comment (prolajumokeoni) [reply index.html:30]: Hi @ONJoseph, currently what I see is the same above kindly make sure you implement the previous changes requested 👍🏿  

- [ ] You can add a `display:none` to menu-links in the mobile version and `display: flex` when needed in the desktop version

image

File: index.html [hunk 4]
```diff
@@ -57,8 +58,313 @@ <h6 class="link-item">
...
+                    <p class="code-title">HTML</p>
+                  </div>
+                </li>
+                <li class="code">
+                  <div class="code-template">
+                    <div class="code-img-c"></div>
+                    <p class="code-title">CSS</p>
+                  </div>
+                </li>
+              </ul>
```
Comment (prolajumokeoni) [inline index.html:298]: Your design             |  Figma Design
:-------------------------:|:-------------------------:
icons  | hgood

- [ ] Kindly add the appropriate icons to the li tags and align to the left, check screenshot above ⬆️

File: index.html [hunk 5]
```diff
@@ -57,8 +58,313 @@ <h6 class="link-item">
...
+          <div class="project-buttom-container">
+            <a href="#" class="project-buttom-b">Get my resume</a>
+          </div>
+        </div>
+        <div class="about-items-b">
+          <div class="about-knowledge-container">
+            <div class="skill-container" id="active">
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Languages</a></li>
```
Comment (prolajumokeoni) [inline index.html:273]: - [ ] Kindly add the  right icon to languages 🤝

File: index.html [hunk 6]
```diff
@@ -57,8 +58,313 @@ <h6 class="link-item">
...
+                    <div class="code-img-c"></div>
+                    <p class="code-title">CSS</p>
+                  </div>
+                </li>
+              </ul>
+            </div>
+            <div class="skill-container">
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Frameworks</a></li>
```
Comment (prolajumokeoni) [inline index.html:303]: - [ ] Kindly add the right icon to frameworks 🤝

File: index.html [hunk 7]
```diff
@@ -57,8 +58,313 @@ <h6 class="link-item">
...
+                </ul>
+              </div>
+              <div class="vector-icon"></div>
+            </div>
+            <div class="skill-container">
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Skills</a></li>
```
Comment (prolajumokeoni) [inline index.html:311]: - [ ] Kindly add the right icon to skills 🤝

File: index.html (hunk 2 above)
Comment (prolajumokeoni) [inline index.html:262]: - [ ] Kindly align all items in the about section to the left 👍🏿
 
image

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @ONJoseph 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
- No linter errors
- Github flow was correctly used
- Good commit messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
153658060-ecd8e5a1-4ead-49a4-98e8-26788543abd4

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**